src/config.js

```diff
@@ -11,6 +11,12 @@ export default {
     },
     baseUrl: __INJECTED_CONFIG.dashboard.apiBaseUrl || 'http://localhost:8000/api'
   },
+  logApi: {
```

Comment:
No need another wrapper `logApi` just for a new endpoint. Just append it under `api.endpoints` like this:

```js
   api: {
     endpoints: {
       status: '/status',
       statusLogs: '/status/logs'
      },
      baseUrl: __INJECTED_CONFIG.dashboard.apiBaseUrl || 'http://localhost:8000/api'
    },
```